### PR TITLE
DROOLS-2827: [DMN Designer] SVG produced for each save operation

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-2827

This PR removes the automatic creation of diagram SVGs when the file is saved from DMN.

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins, please retest this.

**Comment by manstis** [comment]
@triceo Hi.. is there anybody I can have from QE review my PRs whilst @jomarko is on PTO?

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

(Sure this green a few hours ago).

File: kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-project/kie-wb-common-stunner-project-client/src/test/java/org/kie/workbench/common/stunner/project/client/session/EditorSessionCommandsTest.java [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        verify(commands).get(15);
+    }
+
+    @Test
+    public void testGet() {
+        IntStream.range(0, 15).forEach(i -> {
+            editorSessionCommands.get(i);
+            verify(commands).get(i);
+        });
+    }
```
Comment (jomarko) [inline kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-project/kie-wb-common-stunner-project-client/src/test/java/org/kie/workbench/common/stunner/project/client/session/EditorSessionCommandsTest.java]: I think this is the only test method we want to leave here, right?

**Comment by jomarko** [comment]
@manstis please review the testclass I commented above and let me know if this is fine to merge. I finished manual check and everything looks fine.

File: kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-project/kie-wb-common-stunner-project-client/src/test/java/org/kie/workbench/common/stunner/project/client/session/EditorSessionCommandsTest.java (hunk 1 above)
Comment (manstis) [reply kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-project/kie-wb-common-stunner-project-client/src/test/java/org/kie/workbench/common/stunner/project/client/session/EditorSessionCommandsTest.java]: @jomarko IIUC are you proposing we only need the one test ```testGet()```?

I added the other tests to ensure the explicit methods get the correct command too.

Although ```editorSessionCommands.get(0)``` is verified to call ```commands.get(0)``` there is nothing to say (without my other tests) that ```getCutSelectionSessionCommand()``` retrieves ```commands.get(14)``` (for example).

**Pull request merged**